tools/build_all.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+# All the parenthesis around the calls to setup.bash and the build are to
+# prevent the ROS variables from entering the global scope of this script.
+# It seems that the ros1_bridge build will fail if the ros2 environment is
+# sourced before ROS 1, so we need to make sure that no ROS 2 variables are
+# set at the time we start the last step, the ros1_bridge build.
+
+set -e
+
+CWD=`pwd`
+
+# Determin which repos to build. If the ROS 1 and ROS 2 base directories are not
+# present, then don't build
+if [ -d "ros1_dependencies_ws" ]; then
+  ENABLE_ROS1=true
+else
+  ENABLE_ROS1=false
+fi
+if [ -d "ros2_ws" ]; then
+  ENABLE_ROS2=true
+  ROS2_SETUP_FILE=$CWD/ros2_ws/install/setup.sh
+else
+  ENABLE_ROS2=false
+  ROS2_SETUP_FILE=/opt/ros/bouncy/setup.sh
+fi
+
+
+# Build ROS 1 dependencies
+if [ "$ENABLE_ROS1" = true ]; then
+  cd ros1_dependencies_ws
+  (. /opt/ros/kinetic/setup.sh &&
```

Comment:
Similarly can kinetic be changed to $ROS1_DISTRO so that we can run this with Melodic?

Reply:
To change the ros1 version we use most likely requires changing the ros1_dependencies.repos file. What I'll do is create two ros1 repos files, one for kinetic and one for melodic, and those will be the only two we'll support for now.